pkg/virt-handler/migration-source_test.go

```diff
@@ -527,6 +528,10 @@ var _ = Describe("VirtualMachineInstance migration target", func() {
 
 		domain := api.NewMinimalDomainWithUUID("testvmi", vmiTestUUID)
 		domain.Status.Status = api.Running
+		const testIfaceName = "eth0"
```

Comment:
Please consider following up on the negative scenario where netstat.UpdateStatus fails.